src/app/transform/transform.component.ts

```
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { MycustompipePipe } from '../core/pipe/mycustompipe.pipe';
+import { KebabCasePipe } from '../core/pipe/kebab-case.pipe';
+
+@Component({
+  selector: 'app-transform',
+  imports: [MycustompipePipe, KebabCasePipe],
+  templateUrl: './transform.component.html',
+  styleUrl: './transform.component.css'
+})
+export class TransformComponent implements OnInit {
+  constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)